Library/Formula/libodb-qt.rb

```diff
@@ -0,0 +1,71 @@
+class LibodbQt < Formula
+  homepage "http://www.codesynthesis.com/products/odb/"
+  url "http://www.codesynthesis.com/download/odb/2.4/libodb-qt-2.4.0.tar.gz"
+  sha256 "f48c563f653df178866301e4c5069b09b697d98f54dcc9ef8714157002156183"
+
+  depends_on "libodb" => :recommended
+  option "without-libodb", "Don't install libodb"
```

Comment:
Can we just structure it like this please?

```ruby
option "without-ducks", "No ducks installed :("
option "with-extra-ducks", "More ducks for everyone!"

depends_on "ducks" => :recommended
depends_on "extra-ducks" => :optional
```

Merging the two blocks makes it a nightmare to read.